salt/etc-hosts/hosts.jinja

```diff
@@ -20,7 +20,7 @@
 	{%- else %}
 		{%- set ip = addrlist|first %}
 	{%- endif %}
-{{ ip }} {{ master_id }}.{{ pillar['internal_infra_domain'] }}
+{{ ip }} {{ master_id }}.{{ pillar['internal_infra_domain'] }} {{ pillar['api']['server']['external_fqdn'] }}
```

Comment:
I believe this needs to be moved to the 127.0.0.1 entry above? With multiple masters, we'll end up duplicating this name over multiple IPs